src/thunderbots/software/ai/hl/stp/tactic/passer_tactic.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "ai/hl/stp/tactic/tactic.h"
+#include "ai/passing/pass.h"
+
+/**
+ * This tactic is for a robot performing a pass. It should be used in conjunction with
+ * the `ReceiverTactic` in order to complete the pass.
+ *
+ * Note that this tactic does not take into account the time the pass should occur at,
+ * it simply tries to move to the best position to take the pass as fast as possible
+ */
+class PasserTactic : public Tactic
+{
+   public:
+    /**
+     * Creates a new PasserTactic
+     *
+     * @param pass The pass this tactic should try to execute
+     * @param loop_forever Whether or not this Tactic should never complete. If true, the
+     * tactic will be restarted every time it completes
+     */
+    explicit PasserTactic(AI::Passing::Pass pass, bool loop_forever);
```

Comment:
nit: I'm guessing `Pass` has no default constructor? I think it would be a bit better if we didn't have to construct the tactic with a pass. Maybe make this an optional so we don't have to construct it right away?